app/src/main/java/xyz/klinker/blur/launcher3/Launcher.java

```diff
@@ -385,8 +385,17 @@ public void run() {
 
     private RotationPrefChangeHandler mRotationPrefChangeHandler;
 
+
+    private static Launcher sLauncher;
+
+    public static Launcher getInstance() {
```

Comment:
I don't like holding a static `Launcher` here. I made some comments below about why it isn't necessary anyways. We already have an instance of the `Launcher`, in the quick setting bar.

I think that this should be removed